db/migrate/20171128145842_add_chat_id_to_topics.rb

```diff
@@ -0,0 +1,5 @@
+class AddChatIdToTopics < ActiveRecord::Migration[5.1]
+  def change
+    add_column :topics, :chat_id, :integer
```

Comment:
A topic can be in many chats. Here you are adding just one chat per topic.
I think you have to change the association for topic and chats.

belongs_to means that the foreign key is in the table for this class. So belongs_to can ONLY go in the class that holds the foreign key. The foreign key will go to chat (topic_id)
so the association could be like
```
class Chat < ApplicationRecord  
  belongs_to :topic
end
```
And a topic can have many chats